src/Test.Second/LargestTwo.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+
+namespace Test.Second
+{
+    public class LargestTwo
+    {
+        public int[] Largest(int[] arr)
+        {
+            if (arr.Length < 2)
+            {
+                throw new ArgumentException("Array should have at least 2 elements");
+            }
+
+            var first = int.MinValue;
+            var second = int.MinValue;
+            foreach (var num in arr)
+            {
+                if (first < num)
+                {
+                    second = first;
+                    first = num;
+                }
+                else if (second < num)
+                {
+                    second = num;
+                }
+            }
+
+            return new int[] { first, second };
```

Comment:
Is there a better structure to store a result composed only of 2 values?

Reply:
A `ValueTuple`. I've refactored the method to return a `ValueTuple`. I didn't use it the first time around because xunit doesn't support one in `InlineData`